packages/webapp/src/containers/Animals/AddAnimals/types.ts

```diff
@@ -89,7 +89,7 @@ export type Option = {
   [DetailsFields.USE]: ReactSelectOption<number>;
   [DetailsFields.TAG_COLOR]: ReactSelectOption<number>;
   [DetailsFields.TAG_TYPE]: ReactSelectOption<number>;
-  [DetailsFields.ORGANIC_STATUS]: ReactSelectOption<number>;
+  [DetailsFields.ORGANIC_STATUS]: ReactSelectOption<'Non-Organic' | 'Transitional' | 'Organic'>;
```

Comment:
Could consider making `'Non-Organic' | 'Transitional' | 'Organic'` a reusable type -- its used twice here and would be useful for any locations that are going to be added like pasture.